pkgs/by-name/co/cosmic-greeter/package.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rust
+, rustPlatform
+, cmake
+, just
+, pkg-config
+, libxkbcommon
+, fontconfig
+, freetype
+, expat
+, linux-pam
+, wayland
+, which
+, lld
+, util-linuxMinimal
```

```suggestion
, linux-pam
, wayland
```

Fixup args for above review comments